src/components/SingleNote/SingleNote.js

```diff
@@ -33,20 +34,23 @@ export const SingleNote = ({note, setEditModal, isHomePage = false, isTrashPage
     }
    
     return(
-        <div className="single-note margin-xs padding-xs" style={{backgroundColor:color}}>
-            <h2 className="head-md">{title}</h2>
+        <div 
+            className={`single-note margin-xs padding-xs  ${theme === "light" ? "light-theme" : "dark-theme"} `} 
+            style={{backgroundColor:color}}
+        >
+            <h2 className="note-title">{title}</h2>
             <div className="text-sm padding-sm" dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(body) }}></div>
```

Comment:
suggestion: give this a read: https://javascript.plainenglish.io/sanitize-your-html-and-keep-its-style-in-react-1ff9fdd8e3cc

Reply:
Thanks for this suggestion